addons/common/fnc_uniqueUnitItems.sqf

```diff
@@ -10,6 +10,7 @@ Parameters:
     _backpack       - Include items in backpack (Default: true)
     _vest           - Include items in vest     (Default: true)
     _uniform        - Include items in uniform  (Default: true)
+    _assignedItems        - Include assigned items  (Default: true)
```

Comment:
What happened to alignment here?

Reply:
Dedmen just copy pasted a entry from above and changed the names. Dunno why I didn't see that 
(╯°□°）╯︵ ┻━┻